cypress/support/init/leaveInit.ts

```diff
@@ -0,0 +1,32 @@
+import { addEntitlementPayload } from "../API/payload/addEntitlementsPayload";
+import { applyLeavePayload } from "../API/payload/applyLeavePayload";
+import { approveLeaveRequestPayload } from "../API/payload/approveLeaveRequestPayload";
+
+export default class leave {
```

Comment:
PascalCase, and what is the difference between this class and the previous one? each should have a unique name